src/plugins/plugins.js

```diff
@@ -20,70 +20,70 @@
+import ConditionPlugin from './condition/plugin.js';
+import ConditionWidgetPlugin from './conditionWidget/plugin.js';
+import CouchDBSearchFolder from './CouchDBSearchFolder/plugin.js';
+import DefaultRootName from './defaultRootName/plugin.js';
+import DeviceClassifier from './DeviceClassifier/plugin.js';
+import DisplayLayoutPlugin from './displayLayout/plugin.js';
+import FaultManagementPlugin from './faultManagement/FaultManagementPlugin.js';
+import Filters from './filters/plugin.js';
+import FlexibleLayout from './flexibleLayout/plugin.js';
+import FolderView from './folderView/plugin.js';
+import FormActions from './formActions/plugin.js';
+import GaugePlugin from './gauge/GaugePlugin.js';
+import GoToOriginalAction from './goToOriginalAction/plugin.js';
+import Hyperlink from './hyperlink/plugin.js';
+import ImageryPlugin from './imagery/plugin.js';
+import InspectorDataVisualization from './inspectorDataVisualization/plugin.js';
+import InspectorViews from './inspectorViews/plugin.js';
+import ObjectInterceptors from './interceptors/plugin.js';
+import ISOTimeFormat from './ISOTimeFormat/plugin.js';
+import LADTable from './LADTable/plugin.js';
+import LocalStorage from './localStorage/plugin.js';
+import LocalTimeSystem from './localTimeSystem/plugin.js';
+import MyItems from './myItems/plugin.js';
+import NewFolderAction from './newFolderAction/plugin.js';
+import { NotebookPlugin, RestrictedNotebookPlugin } from './notebook/plugin.js';
+import NotificationIndicator from './notificationIndicator/plugin.js';
+import ObjectMigration from './objectMigration/plugin.js';
+import OpenInNewTabAction from './openInNewTabAction/plugin.js';
+import OperatorStatus from './operatorStatus/plugin.js';
+import PerformanceIndicator from './performanceIndicator/plugin.js';
+import CouchDBPlugin from './persistence/couch/plugin.js';
+import PlanLayout from './plan/plugin.js';
+import PlotPlugin from './plot/plugin.js';
+import RemoteClock from './remoteClock/plugin.js';
+import StaticRootPlugin from './staticRootPlugin/plugin.js';
+import SummaryWidget from './summaryWidget/plugin.js';
+import Tabs from './tabs/plugin.js';
+import TelemetryMean from './telemetryMean/plugin.js';
+import TelemetryTablePlugin from './telemetryTable/plugin.js';
+import Espresso from './themes/espresso.js';
+import Snow from './themes/snow.js';
+import TimeConductorPlugin from './timeConductor/plugin.js';
+import Timeline from './timeline/plugin.js';
+import TimeList from './timelist/plugin.js';
+import Timer from './timer/plugin.js';
+import URLIndicatorPlugin from './URLIndicatorPlugin/URLIndicatorPlugin.js';
+import URLTimeSettingsSynchronizer from './URLTimeSettingsSynchronizer/plugin.js';
+import UserIndicator from './userIndicator/plugin.js';
+import UTCTimeSystem from './utcTimeSystem/plugin.js';
+import ViewDatumAction from './viewDatumAction/plugin.js';
+import ViewLargeAction from './viewLargeAction/plugin.js';
+import WebPagePlugin from './webPage/plugin.js';
```

Comment:
i'm  just going to assume these are all correct 😅

Reply:
the diff helps 👍 - those `.js` transformations were done with regex